### DISCUSSION TITLE
关于starcoin的启动的问题，万分感谢！！！

### DISCUSSION THREAD

**Discussion by cl8600** [NonceGeek/Move-Camp#229] [Q&A]
1.目前为止，已经通过docker已经配好了starcoin的容器并且运行成功。如图1
![图片](https://user-images.githubusercontent.com/96363155/211599444-0a2d95f4-4867-43d9-af87-2bc32145b0c6.png)

2。并且在vscode上通过SSH也联通了我的虚拟机。如图2
![图片](https://user-images.githubusercontent.com/96363155/211599774-09844ea3-41bc-493b-89ae-6b0b2596a42c.png)

3.下载了一个项目（地址https://github.com/v1xingyue/movegroup），导入vscode并安装好starcoinIDE之后，尝试运行unittest（如图3），运行失败并在output处报错了（如图4），另外，当时报错的同时也直接退掉了ssh。
![图片](https://user-images.githubusercontent.com/96363155/211601924-2d1401b9-acb1-4ff8-966e-6a6dfc2ccace.png)

4.然后重新连接上ssh之后再重复运行unit test就没有反应了。（binary一类的依赖也下载好了，初次运行的时候vscode自动下载的）
![图片](https://user-images.githubusercontent.com/96363155/211601055-ef6a20ba-17fd-4362-871e-22e97b3e239f.png)

目前进行到了这个阶段，主要是想确认一下是否还有欠缺的步骤，然后还有后续步骤需要做的，请maintainer和大神们指点😁

**Comment by mason0510** [comment]
检查下节点是否正常运行

**Comment by v1xingyue** [comment]
你那个报错应该是 本地的 move-analyzer 没有装好。
如果是 连接 的remote 应该是 去  remote 里边去安装配置。 

安装方法可以针对如下的插件文档，本地安装配置一下: 
https://marketplace.visualstudio.com/items?itemName=move.move-analyzer

**Reply by cl8600** [reply]
好的好的，非常感谢！我等先把课程都补完之后再继续看看怎么回事
